Week-1/Homework/mandatory/3-groceries.js

```diff
@@ -1,3 +1,15 @@
+/* ======= TESTS - DO NOT MODIFY =====
```

Comment:
Same as the above - could this be moved into a JS file that can be included here? Also reduced duplicated code